tests/common/include/tests/common/tools/intersectors/helix_cylinder_intersector.hpp

```diff
@@ -98,13 +98,13 @@ struct helix_cylinder_intersector
         // Note: the default path length might be smaller than either solution
         switch (qe.solutions()) {
             case 2:
-                paths[1] = 0.95f * qe.larger();
+                paths[1] = qe.larger();
```

Comment:
So this forces to trigger the newton method but I guess we really don't have to :thinking:

Reply:
True, since I moved to use the actual cylinder intersector for the initial estimate now. Then we will need to switch to the ray plane intersector in the ```helix_plane_intersector```. That would be a good idea anyway